### ISSUE TITLE
Allow fields to be applied to all new objects by using a single configuration value

### ISSUE THREAD

**Issue by mjordan** [mjordan/islandora_workbench#23]
If a value in a CSV field is the same for all new nodes (task = create), it might be useful to allow this field:value pair to be defined once in the configuration file. Some examples are #22, and #12. The logic would be if a field name: value pair is present in the config file, apply that value to all new nodes (task = create). We would probably need to create a nested map in the config file, e.g.:

```yaml
node_fields:
  field_member_of: 12
  content_type: islandora_object
```
In this example, all of the newly created nodes would be islandora_object nodes with a `field_member_of` value of 12.

**Comment by mjordan** [comment]
This would be very useful when creating nodes using the `create_from_files` task, where there is no CSV file.

**Comment by mjordan** [comment]
Resolved with 7cc5624af98d90ac9618ffefacfcdde05dadc867, including docs in the README and a single integration test.
